tests/features/data-visibility-and-review.feature

```diff
@@ -16,78 +16,107 @@ Feature: Manage data visibility separately from review and approval in the backe
 
   Background:
     Given a functioning api
-    And the dataset model has a visibility field that supports "internal" and "public"
-    And the dataset model has a review_status field that supports "provisional" and "approved"
-    And visibility and review_status are required fields when creating new datasets
-    And new datasets must use visibility and review_status as the source of truth
+    And all database models have a visibility field that supports "internal" and "public"
+    And all database models have a review_status field that supports "provisional" and "approved"
+    And visibility and review_status are required fields when creating new records
+    # hard to test?

```

Comment:
I don't think it'll be too hard to test. We can just make sure in `BaseCreateSchema` that those fields are always required for new records, so if they are not submitted Pydantic will raise errors.